README.md

```diff
@@ -13,23 +13,23 @@ A widget to visualize and edit atomistic structures in Jupyter Notebook. It uses
 Features:
 
 - Model: space-filling, ball-stick, polyhedral.
-- Supported File type: cif, xyz.
+- Supported File types: cif, xyz.
```

Comment:
```suggestion
- Supports importing data from ASE and Pymatgen.
```

For the moment, the widget itself does not support reading data from the files directly. We can add this later, since the WEAS (javascript) code can read cif, XYZ, and cube files.